src/soc/ibm/power9/istep_8_1.c

```diff
@@ -0,0 +1,337 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#include <arch/io.h>
+#include <stdint.h>
+#include <string.h>
+#include <console/console.h>
+#include <cpu/power/istep_8.h>
+#include <cpu/power/mvpd.h>
+#include <cpu/power/scom.h>
+
+#include "fsi.h"
+#include "homer.h"
```

Comment:
This is only for `IS_*_FUNCTIONAL`? If this is the case then it should probably be put into another, smaller header. Probably together with `get_available_cores()`, and this function should be used in homer.c (this may be done later, when working on secondary HOMER & OCC stuff). The way I did it seems too hacky, and may not work for second CPU anyway.

Reply:
> This is only for `IS_*_FUNCTIONAL`?

Yes. Macros for checks and `MAX_*_PER_CHIP` constants are two common reasons for including `homer.h`.